woocommerce/woocommerce-ios
# Remove remaining localization scripts made unused by new tooling

### Description

Removes three scripts that the new localization tooling, #6936, superseded.

### Testing instructions

Nothing to test, these scripts are no longer called, in particular because of 0b48848 and c991e2a which replaced the call to the legacy [`ios_update_metadata` action](https://github.com/wordpress-mobile/release-toolkit/blob/d80471182e80d151773153536bcfc590e83a962c/lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_update_metadata.rb), and 066b80acb which did the same for [`ios_localize_project`](https://github.com/wordpress-mobile/release-toolkit/blob/d80471182e80d151773153536bcfc590e83a962c/lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_localize_project.rb#L9). Those actions, in turn, called [the helper methods directly resulting in those scripts running](https://github.com/wordpress-mobile/release-toolkit/blob/d80471182e80d151773153536bcfc590e83a962c/lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb#L31-L64).

---
